extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -463,18 +474,92 @@ export class GoogleSubscriptionsPlatform {
    * @return {boolean}
    */
   isPingbackEnabled() {
-    return false;
+    // Only enable pingback if useful deferred account creation.
+    return (this.serviceConfig_["hasAssociatedAccountUrl"] &&
+            this.serviceConfig_["createOrUpdateAccountUrl"]);
   }
 
   /** @override */
   pingbackReturnsAllEntitlements() {
-    return false;
+    return true;
+  }
+
+  /** @return {!Promise<{found: boolean}>} */
+  hasAssociatedUserAccount_(selectedEntitlement) {
+    const hasAssociatedAccountUrl = /** @type {string} */ (devAssert(
+      this.serviceConfig_["hasAssociatedAccountUrl"] ,
+      'hasAssociatedAccountUrl is null'
+    ));
+
+    const promise = this.urlBuilder_.buildUrl(
+      hasAssociatedAccountUrl,
+      /* useAuthData */ true
+    );
+    return promise.then(url => {
+      return this.xhr_.fetchJson(url, {
+        method: 'POST',
+        credentials: 'include',
+        body: this.serviceAdapter_.stringifyForPingback(selectedEntitlement),
+      });
+    });
   }
 
   /**
-   * Performs the pingback to the subscription platform
-   */
-  pingback() {}
+   * @param {!DeferredAccountCreationResponse} deferredAccountResponse
+   * @return {!Promise<{}>}
+   * */
+  createOrUpdateAccount_(deferredAccountResponse) {
+    const createOrUpdateAccountUrl = /** @type {string} */ (devAssert(
+      this.serviceConfig_["createOrUpdateAccountUrl"] ,
+      'createOrUpdateAccountUrl is null'
+    ));
+
+    const promise = this.urlBuilder_.buildUrl(
+      createOrUpdateAccountUrl,
+      /* useAuthData */ true
+    );
+    console.log('DeferredAccountCreationResponse')
+    return promise.then(url => {
+      return this.xhr_.fetchJson(url, {
+        method: 'POST',
+        credentials: 'include',
+        body: this.serviceAdapter_.stringifyForPingback({
+          entitlements: deferredAccountResponse.entitlements,
+          userData: deferredAccountResponse.userData,
+          purchaseDataList: deferredAccountResponse.purchaseDataList,
+        }),
+      });
+    });
+  }
+
+  /**
+   * @override
+   * @param {?./entitlement.Entitlement} entitlements
+   * @return {!Promise|undefined}*/
+  pingback(entitlements) {
+    if (!this.isPingbackEnabled()) {
+      return;
+    }
+    return this.hasAssociatedUserAccount_(entitlements).then(async result => {
+      console.log(`result ${result.json()}`)
+      const {found} = await result.json();
+      console.log(`found ${found}`)
+      if (found) {
+        this.runtime_.completeDeferredAccountCreation({
+          entitlements
+        }).then(response => {
+          console.log(`response ${response}`);
+          this.createOrUpdateAccount_(response).then(() => {
+            console.log(`response ${response.complete}`);
```

Comment:
This would be response from line #550?

Reply:
It's the response from the deferred account creation method